components/server/src/iam/iam-session-app.ts

```diff
@@ -42,9 +43,21 @@ export class IamSessionApp {
             app.use(middleware);
         });
 
+        // Use RequestContext
+        app.use((req, res, next) => {
+            runWithRequestContext(
+                {
+                    requestKind: "iam-session-app",
+                    requestMethod: req.path,
+                    signal: new AbortController().signal,
+                },
+                () => next(),
+            );
+        });
+
         app.post("/session", async (req: express.Request, res: express.Response) => {
             try {
-                const result = await this.doCreateSession(req, res);
+                const result = await runWithSubjectId(SYSTEM_USER, async () => this.doCreateSession(req, res));
```

Comment:
I don't understand why we don't need anymore runWithSubjectId for SYSTEM_USER?

Reply:
was an intent that it should have been passed as past of request context already above?